docs/using-ngrok-with/googleColab.md

```diff
@@ -4,4 +4,4 @@ title: Google Colab
 
 # Using ngrok with Google Colab
 
-If you're looking to run the ngrok agent into your Google Colab project, you can leverage the [pyngrok project](https://pyngrok.readthedocs.io/en/latest/integrations.html#google-colaboratory) to start a tunnel in your project.
+If you're looking to run the ngrok agent into your Google Colab project, you can leverage the [ngrok Python SDK](https://github.com/ngrok/ngrok-python) to start a tunnel in your project.
```

Comment:
we will need an example for google colab either here or in the repo if we want to update this IMO.

Reply:
added https://github.com/ngrok/ngrok-python/issues/131